validation/src/main/scala/hmda/validation/rules/lar/validity/V230.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ LoanApplicationRegister }
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+
+object V230 extends EditCheck[LoanApplicationRegister] {
+
+  val occupancyStatusTypes = List(1, 2, 3)
+
+  def apply(lar: LoanApplicationRegister): Result = {
+    lar.loan.occupancy is containedIn(occupancyStatusTypes)
+  }
+
+  def name = "V230"
```

Comment:
My understanding was that the `override` modifier is optional when extending an abstract class. Is it really necessary, or is it just stylistically preferable? (Just curious)